docs/editors/emacs.md

```diff
@@ -33,8 +33,11 @@ To use Metals in Emacs, place this snippet in your Emacs configuration (for exam
 (require 'use-package)
 
 ;; Enable defer and ensure by default for use-package
+;; Keep auto-save/backup files separate from source code:  https://github.com/scalameta/metals/issues/1027
 (setq use-package-always-defer t
-      use-package-always-ensure t)
+      use-package-always-ensure t
+      backup-directory-alist `((".*" . ,temporary-file-directory))
+      auto-save-file-transforms `((".*" ,temporary-file-directory t)))
```

Comment:
hey, I think you mean `auto-save-file-name-transforms`, because `auto-save-file-transforms` does not seem to be defined in emacs.

Reply:
Ugh. You are correct, the variable name should be auto-save-file-name-transforms. I had it correct in my personal emacs config and then messed it up putting the changes into the metals repo. Thank you for catching my boneheaded mistake.